src/main/java/core/basesyntax/service/DataReader.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dto.Fruit;
+import core.basesyntax.dto.Operation;
+import core.basesyntax.exceptions.IllegalDataException;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReader {
+    private static final int TYPE_FIELD = 0;
+    private static final int FRUIT_FIELD = 1;
+    private static final int QUANTITY_FIELD = 2;
+    private static final int SIZE = 3;
+
+    private String fileName;
+
+    public DataReader(String fileName) {
+        this.fileName = fileName;
+    }
+
+    public static List<Operation> readFromFile(String fileName) throws IOException {
+        List<Operation> listOfOperations = new ArrayList<>();
```

Comment:
Ideally returning a list of <Operation> would be parser's job
readFromFile should just return data as it is.. List<String>